tests/unit/services/address-register-test.js

```diff
@@ -0,0 +1,12 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
```

Comment:
maybe can remove that test file?

Reply:
I would write tests for it instead 😄.

Reply:
it's good for an ipi to write tests :p

Reply:
I think something like msw might be nice to mock the request responses: https://mswjs.io/

Ember also has a built-in mocking setup, but it's not documented very good.